spec/core/block/schemas/BlockMetaDTO.yml

```diff
@@ -16,9 +18,23 @@ properties:
     type: array
     items:
       $ref: "../../../schemas/Hash256.yml"
-  numTransactions:
+  totalTransactionsCount:
     type: integer
-    example: 0
-  numStatements:
+    description: >
+      Total number of [transactions](https://docs.symbolplatform.com/concepts/transaction.html)
+      present in this block, including *embedded* transactions
+      (i.e. transactions contained within aggregate transactions).
```

Comment:
i still think this is too verbose for here

i'd go with stuff like:

_Total number of transactions that were confirmed in this block, including *embedded* transactions_

links are not necessary in here imo, but your choice

Reply:
Links do not hurt expert readers (just don't click on them), and they definitely help newcomers.

Regarding verbosity, I have the feeling most of the current docs can only be understood by experts and are not very kind to newcomers. If we are to extend our user base we should change that...

In this case it's only a parenthesis with 5 words so I don't think it's too much burden for experts :)